datadog_builder/update.py

```diff
@@ -80,7 +82,10 @@ def _create_monitor(client, args, monitor):
     if args.dry_run:
         LOG.warn("Create new monitor %(name)s", monitor)
     else:
-        client.create_monitor(monitor)
+        try:
+            client.create_monitor(monitor)
+        except HTTPError:
+            LOG.error("Monitor %(name)s failed to create", monitor)
```

Comment:
If you use LOG.exception here it will still log as error but you'll get traceback in the logs automatically